h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -2987,3 +2987,20 @@ def check_ignore_cols_automl(models,names,x,y):
         else:
             assert set(h2o.get_model(model).params["ignored_columns"]["actual"]) == set(names) - {y} - set(x), \
                 "ignored columns are not honored for model " + model
+
+def check_sorted_2_columns(frame1, sorted_column_indices, prob=0.5):
+    for colInd in sorted_column_indices:
+        for rowInd in range(0, frame1.nrow-1):
+            if (random.uniform(0.0,1.0) < prob):
+                if colInd == sorted_column_indices[0]:
```

Comment:
I don't understand this, what if sorted_column_indices = [1, 2], for colInd = 1 it will access frame1[rowInd, 0] - isn't that a wrong column of the frame?

Reply:
You are right, I should use colInd.